python/src/iceberg/utils/transform_util.py

```diff
@@ -0,0 +1,110 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I'm wondering if it's worth splitting this out into a separate file like this. It's less than 100 lines of code and only used by `transforms.py`, is it better to just place it at the top of that file?